src/c++/perf_analyzer/genai-perf/tests/test_llm_inputs.py

```diff
@@ -368,8 +368,10 @@ def test_synthetic_to_openai_completions(self, default_tokenizer):
             (OutputFormat.VLLM),
         ],
     )
-    def test_llm_inputs_extra_inputs(self, default_tokenizer, output_format) -> None:
-        request_inputs = {"additional_key": "additional_value"}
+    def test_extra_inputs(self, default_tokenizer, output_format) -> None:
```

Comment:
do we also check if the default max tokens is set properly when we don't specify extra inputs?

Reply:
Thanks for the question! I wrote a quick unit test to check it that I did not include in the PR. Just pushed it in case we want it.